frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeBreakoutColumnList.tsx

```diff
@@ -33,7 +32,8 @@ export const SummarizeBreakoutColumnList = ({
     spacing="0"
     {...containerProps}
   >
-    <SectionTitle>{t`Group by`}</SectionTitle>
+    <Title order={5} fw={900}>{t`Group by`}</Title>
```

Comment:
the difference here is that `SectionTitle` was using `h3` whose font-size was `16.38px` (`1.17em`). This changes it to `16px` and keeps the same font weight.